services/ai-navigator-app/0.2.8/ai-navigator-app.yaml

```diff
@@ -60,7 +63,7 @@ spec:
       serviceAccountName: ai-navigator-app
       containers:
         - name: ai-navigator-app
-          image: mesosphere/ai-navigator-app:v0.1.1
+          image: mesosphere/ai-navigator-app:v0.2.0
```

Comment:
I don't think we can backport the agent - it might not work well with the version of Flux shipped with 2.12. We don't need to patch the agent for cluster-info-api, so I would suggest reverting these changes